runtime/parachains/src/configuration.rs

```diff
@@ -278,6 +277,28 @@ pub trait WeightInfo {
 	fn set_hrmp_open_request_ttl() -> Weight;
 }
 
+pub struct TestWeightInfo;
+impl WeightInfo for TestWeightInfo {
+	fn set_config_with_block_number() -> Weight {
+		0
+	}
+	fn set_config_with_u32() -> Weight {
+		0
+	}
+	fn set_config_with_option_u32() -> Weight {
+		0
+	}
+	fn set_config_with_weight() -> Weight {
+		0
+	}
+	fn set_config_with_balance() -> Weight {
+		0
+	}
+	fn set_hrmp_open_request_ttl() -> Weight {
+		0
```

Comment:
Mind if we change these `0`s to `BlockWeight::default().max_block`?

Reply:
1f6923dc84f02cba54c6ec52539ed0ff71935710